JM/CF/Scripts/3_Game/CommunityFramework/XML/CF_XML_Callback.c

```diff
@@ -0,0 +1,10 @@
+typedef CF_XML_Callback XMLCallback;
```

Comment:
Not sure why git did not detect this as a file location move. Please validate that there were no accidental changes.